src/app/search/search.component.html

```diff
@@ -72,7 +72,13 @@
           (changedDebounced)="handleChangedDebounced($event)"
           (submitted)="handleSubmitted($event)"
         ></app-basic-search>
-        <mat-expansion-panel *ngFor="let key of getKeys(orderedBuckets)" #bucket expanded>
+        <mat-expansion-panel
+          *ngFor="let key of getKeys(orderedBuckets)"
+          #bucket
```

Comment:
What is `#bucket` for? It was there before, but just curious.

Reply:
Good point, I have no idea... can't find any reference to it and removing it doesn't seem to do anything.  Looks like it was introduced [here](https://github.com/dockstore/dockstore-ui2/pull/48/files#diff-4b285e04baef6b0ed18c0c0501ca161ba52f82e37927b3f22689840438778c26R30) in 2017. I'm gonna remove it